smarts/core/smarts.py

```diff
@@ -267,6 +272,9 @@ def reset(self, scenario: Scenario):
         observations, _, _, _ = self._agent_manager.observe(self)
         observations_for_ego = self._agent_manager.reset_agents(observations)
 
+        # Visualization
+        self._try_emit_visdom_obs(observations)
```

Comment:
If we are doing a visualisation here, is there any reason why envision should not also try?

Reply:
We could, out of scope for the PR at this time. Some minor work would need to be required to support running Envision w/ missing fields (e.g. no scores or rewards).